Rubberduck.Interaction/IMessageBox.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using Forms = System.Windows.Forms;
+
+namespace Rubberduck.Interaction
+{
+    public interface IMessageBox
+    {
+        void Message(string text);
```

Comment:
Should this have a caption parameter?

Reply:
I don't see much of a benefit, honestly. And I am somewhat hard-pressed to come up with a good default, soo ...

Reply:
I mention it only be because all the other methods have a caption, this was the odd one out.